docs-markdown/src/controllers/link-controller.ts

```diff
@@ -84,40 +84,51 @@ export async function collapseRelativeLinksInFolder(uri: Uri) {
 }
 
 export async function collapseRelativeLinks() {
-	await collapseRelativeLinksForEditor(window.activeTextEditor);
-}
-
-export async function collapseRelativeLinksForEditor(editor: TextEditor): Promise<number> {
+	const editor = window.activeTextEditor;
 	if (!editor) {
 		noActiveEditorMessage();
 		return;
 	}
 
-	const content = editor.document.getText();
+	await collapseRelativeLinksForEditor(editor.document, editor);
+}
+
+export async function collapseRelativeLinksForEditor(document : TextDocument, editor: TextEditor): Promise<number> {
+	const content = document.getText();
 	if (!content) {
 		return;
 	}
 
-	const directory = dirname(editor.document.fileName);
-	const tempReplacements = findReplacements(editor.document, content, null, linkRegex);
+	const directory = dirname(document.fileName);
+	const tempReplacements = findReplacements(document, content, null, linkRegex);
+	if (!tempReplacements || tempReplacements.length === 0) {
+		return 0;
```

Comment:
Use 4x spaces, rather than tabs.

Reply:
VS Code automatically chose the indentation, which means... ;)

Reply:
Yea, we are using the same eslint config as the docs-ui team so we are using tabs of size 2 instead of 4 spaces.